bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorageWithDirectEntryLoggerTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.bookie.storage.ldb;
+
+import org.junit.Before;
+
+/**
+ * Unit test for {@link DbLedgerStorage} with directIO entrylogger.
+ */
+public class DbLedgerStorageWithDirectEntryLoggerTest extends DbLedgerStorageTest {
+
+    @Override
+    @Before
+    public void setup() throws Exception {
+        super.setup();
+        conf.setProperty("dbStorage_directIOEntryLogger", true);
```

Comment:
Is it possible to add some assertions in the tests about the fact that we really loaded this implementation?
The tests may pass because we aren't using the new implementation

Reply:
done